### PR TITLE
golint: fix warnings for pkg/events/events.go

### PR DESCRIPTION
**Pull request by aalemayhu**
Fixes the following warnings

pkg/events/events.go:22:2: exported const IdentityAdd should have comment (or a comment on this block) or be unexported
pkg/events/events.go:27:6: exported type EventType should have comment or be unexported
pkg/events/events.go:29:6: exported type Event should have comment or be unexported
pkg/events/events.go:35:1: exported function NewEvent should have comment or be unexported

Related-to: #153 (Resolve golint warnings)
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/cilium/cilium/pull/497%23discussion_r109157015%22%2C%20%22https%3A//github.com/cilium/cilium/pull/497%23discussion_r109158181%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%209c5e6d0c5f0740649fb2e50b385735d47833567a%20pkg/events/events.go%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cilium/cilium/pull/497%23discussion_r109157015%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20work%20with%20%40aanm%20to%20make%20this%20a%20bit%20more%20descriptive%3F%22%2C%20%22created_at%22%3A%20%222017-03-31T13%3A18%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/1417913%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tgraf%22%7D%7D%2C%20%7B%22body%22%3A%20%22Will%20wait%20for%20feedback%20from%20%40aanm%20%20and%20reword%20later.%5Cr%5Cn%5Cr%5CnThanks.%22%2C%20%22created_at%22%3A%20%222017-03-31T13%3A24%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/925044%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/scanf%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20pkg/events/events.go%3AL19-44%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 9c5e6d0c5f0740649fb2e50b385735d47833567a pkg/events/events.go 4'></a> &nbsp;<b><a href='https://github.com/cilium/cilium/pull/497#discussion_r109157015'>File: pkg/events/events.go:L19-44</a></b>
- <a href='https://github.com/tgraf'></a> Can you work with @aanm to make this a bit more descriptive?
- <a href='https://github.com/scanf'></a> Will wait for feedback from @aanm  and reword later.
Thanks.

<a href='https://www.codereviewhub.com/cilium/cilium/pull/497?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cilium/cilium/pull/497?approve=1'></a>&nbsp;<a href='https://github.com/cilium/cilium/pull/497'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by aanm** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cilium/cilium/pull/497'>Refresh</a>.<a href='#crh-update-none'></a>

File: pkg/events/events.go [hunk 1]
```diff
@@ -19,19 +19,25 @@ import (
 )
 
 const (
+        // IdentityAdd the identity creation event.
```
Comment (tgraf) [inline pkg/events/events.go]: Can you work with @aanm to make this a bit more descriptive?

**Review by tgraf — CHANGES_REQUESTED** [review]

File: pkg/events/events.go (hunk 1 above)
Comment (aalemayhu) [reply pkg/events/events.go]: Will wait for feedback from @aanm  and reword later.

Thanks.

**Comment by aalemayhu** [comment]
FYI, @aanm  has kindly provided me some feedback and I have started incorporating them. Being in a different timezone and wanting to attend netdev. This might be delayed a little bit.

Thanks.

**Comment by aalemayhu** [comment]
Please let me know if anything more should be reworded, thanks.

**Review by tgraf — APPROVED** [review]

**Pull request merged**